tests/sql/test_exprs.py

```diff
@@ -92,3 +92,15 @@ def test_between():
 
     expected = df.filter(col("integers").between(1, 4)).collect().to_pydict()
     assert actual == expected
+
+
+def test_is_in():
```

Comment:
I suggest covering some more edge cases if applicable. 

here are some claude thought. not sure they are applicable or not though. mostly related to Nones/Nulls, etc

```python
def test_is_in_edge_cases():
    df = daft.from_pydict({
        "nums": [1, 2, 3, None, 4, 5],
        "strs": ["a", "b", None, "c", "d", "e"],
    })

    # Test with NULL values in the column
    actual = daft.sql("SELECT * FROM df WHERE strs IN ('a', 'b')").collect().to_pydict()
    expected = df.filter(col("strs").is_in(["a", "b"])).collect().to_pydict()
    assert actual == expected

    # Test with empty IN list
    actual = daft.sql("SELECT * FROM df WHERE nums IN ()").collect().to_pydict()
    expected = df.filter(col("nums").is_in([])).collect().to_pydict()
    assert actual == expected

    # Test with numbers and NULL in IN list
    actual = daft.sql("SELECT * FROM df WHERE nums IN (1, NULL, 3)").collect().to_pydict()
    expected = df.filter(col("nums").is_in([1, None, 3])).collect().to_pydict()
    assert actual == expected

    # Test with single value
    actual = daft.sql("SELECT * FROM df WHERE nums IN (1)").collect().to_pydict()
    expected = df.filter(col("nums").is_in([1])).collect().to_pydict()
    assert actual == expected

def test_is_in_type_handling():
    df = daft.from_pydict({
        "mixed": [1, "2", 3.0, True, None, "5"],
    })

    # Test with mixed types in the IN list
    actual = daft.sql("SELECT * FROM df WHERE mixed IN (1, '2', 3.0)").collect().to_pydict()
    expected = df.filter(col("mixed").is_in([1, "2", 3.0])).collect().to_pydict()
    assert actual == expected

def test_is_in_with_expressions():
    df = daft.from_pydict({
        "nums": [1, 2, 3, 4, 5],
    })
    
    # Test IN with arithmetic expressions
    actual = daft.sql("SELECT * FROM df WHERE nums + 1 IN (2, 3, 4)").collect().to_pydict()
    expected = df.filter((col("nums") + 1).is_in([2, 3, 4])).collect().to_pydict()
    assert actual == expected
```